tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -89,6 +89,12 @@ class WskBasicUsageTests
         }
     }
 
+    it should "show help and usage info using the default language" in {
+        val env = Map("LANG" -> "de_DE")
+        // Call will fail with exit code 2 if language not supported
```

Comment:
@dubeejw - If the locale is not supported, we should log the warning and fall back to English.